### ISSUE TITLE
modify recognition training configs (img_aug)

### ISSUE THREAD

**Issue by letsyookyung** [PaddlePaddle/PaddleOCR#5761]
Hello, I want to use img aug to train recognition model. 
However I need help about how to modify **Train part** below. 

    Global:
      use_gpu: True
      epoch_num: 500
      log_smooth_window: 20
      print_batch_step: 10
      save_model_dir: ./train_output/general_rec/rec/try5
      save_epoch_step: 1
      # evaluation is run every 5000 iterations after the 4000th iteration
      eval_batch_step: [0, 2000]
      # if pretrained_model is saved in static mode, load_static_weights must set to True
      cal_metric_during_train: True
      pretrained_model: 'C:/workspace/ocr_training_2_2.4/pretrained_models/rec/korean_mobile_v2.0_rec_train/best_accuracy'
      # checkpoints: ./train_checkpoints/general_rec/rec/best_accuracy
      save_inference_dir: 
      use_visualdl: False
      infer_img:
      # for data or label process
      character_dict_path: ppocr/utils/hospital/new_korean_dict.txt
      max_text_length: 35
      infer_mode: False
      use_space_char: False
    
    
    Optimizer:
      name: Adam
      beta1: 0.9
      beta2: 0.999
      lr:
        name: Cosine
        learning_rate: 0.001
      regularizer:
        name: 'L2'
        factor: 0.00001
    
    Architecture:
      model_type: rec
      algorithm: CRNN
      Transform:
      Backbone:
        name: MobileNetV3
        scale: 0.5
        model_name: small
        small_stride: [1, 2, 2, 2]
      Neck:
        name: SequenceEncoder
        encoder_type: rnn
        hidden_size: 48
      Head:
        name: CTCHead
        fc_decay: 0.00001
    
    Loss:
      name: CTCLoss
    
    PostProcess:
      name: CTCLabelDecode
    
    Metric:
      name: RecMetric
      main_indicator: acc
    
    **Train:
      dataset:
        name: SimpleDataSet
        data_dir: 'E:/workspace/general_rec/train'
        label_file_list: ["E:/workspace/general_rec/rec_gt_train.txt"]
        duplication : 1
        transforms:
          - DecodeImage: # load image
              img_mode: BGR
              channel_first: False
              
          _- RecAug: 
              use_tia : True
              aug_prob: 0.2
          - Config :_ 
    
          - CTCLabelEncode: # Class handling label
          - RecResizeImg:
              image_shape: [3, 32, 320]
          - KeepKeys:
              keep_keys: ['image', 'label', 'length'] # dataloader will return list in this order
      loader:
        shuffle: True
        batch_size_per_card: 256
        drop_last: True
        num_workers: 8**
    
    Eval:
      dataset:
        name: SimpleDataSet
        data_dir: 'E:/workspace/general_rec/train'
        label_file_list: ["E:/workspace/general_rec/rec_gt_test.txt"]
        transforms:
          - DecodeImage: # load image
              img_mode: BGR
              channel_first: False
          - CTCLabelEncode: # Class handling label
          - RecResizeImg:
              image_shape: [3, 32, 320]
          - KeepKeys:
              keep_keys: ['image', 'label', 'length'] # dataloader will return list in this order
      loader:
        shuffle: False
        drop_last: False
        batch_size_per_card: 4
        num_workers: 8

I referred to recognition image augmentation information from **ppocr/data/imaug/rec_img_aug.py,** 
I know how to use this part, 

      -RecAug: 
          use_tia : True
          aug_prob: 0.2

how about using Config class in rec_img_aug.py ? 
I guess Config class probably supports more diverse augmentations, so please help me with using it.

**Comment by letsyookyung** [comment]
I actually solve it!

**Comment by Rai-Nabin** [comment]
> I actually solve it!

How did you solve it?

**Comment by gavinfaimdata** [comment]
> I actually solve it!

How?

**Comment by Aminfaraji** [comment]
> I actually solve it!

how solve it?
